test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionController::TestCase
+  def setup
+    @user = users(:jim)
+  end
+
+  test 'should get show' do
```

Comment:
Drop `should` here, maybe `test '#show returns success'` or something like that.